src/test/countComments.test.js

```diff
@@ -0,0 +1,45 @@
+import { count } from '../modules/displaycomments.js';
+
+describe('count function', () => {
+  let countElement;
+  // Mock getElementById
+  const mockDocument = {
+    getElementById: jest.fn().mockReturnValue(countElement),
+  };
+
+  // Assign the mock document to global object
+  global.document = mockDocument;
```

Comment:
- [ ] Your test cases are failing because of this line. kindly consider fixing this so that all your test cases pass.
result:
<img width="692" alt="Screenshot 2023-06-28 at 11 32 30 PM" src="https://github.com/Mike111222/theMoviesMaze/assets/96683843/442617a0-72e9-43d9-8624-d190d81418e5">

Reply:
Hey @Zuhaib042, Thank you so much for pointing out the change 👍 . I have implemented a few changes and it is passing now. 
![test](https://github.com/Mike111222/theMoviesMaze/assets/61277612/50ec4ab7-5b03-492e-b18c-be9a812823a3)
Hopefully, my project will get approved in the next review. 😊

Reply:
-  Well resolved 👍🏽